### ISSUE TITLE
Ffmpeg install path

### ISSUE THREAD

**Issue by szaimen** [PaulLereverend/NextcloudVideo_Converter#18]
Thank you for your great app!

I would like to use your app with my nextcloud snap installation. But for doing so, I have to configure the path where your app is expecting ffmpeg is installed. How can I do that? Is there possibly an option inside the admin settings to configure that?

Thank you for your help!

**Comment by PaulLereverend** [comment]
By default, the app is looking for ffmpeg on the $PATH. 

Depending on your operating system you can add your ffmpeg location on this variable.

**Comment by szaimen** [comment]
Thank you for your answer.
I think I need a little bit more input on how to exactly do that. 
Maybe you have a link for me with instructions, because I am unfortunately a noob in this topic!
I am using ubuntu 18.04 LTS.

**Comment by fishscene** [comment]
The proposed solution will not work with Nextcloud SNAP. SNAP isolates Nextcloud so that it cannot access any of the host files/programs/etc... Kyrofa proposed a possible workaround:
https://github.com/nextcloud/nextcloud-snap/issues/1046#issuecomment-509676968

**Comment by szaimen** [comment]
Yeah I know. But if i could configure the path to where your app is looking for ffmpeg, I could probably create a symlink from the ffmpeg folder to a path the snap has access to.
Thats the idea.

**Comment by PaulLereverend** [comment]
Hello,
There might be another issue. Ffmpeg will not be able to write on the data folder because of the access rights.

**Comment by Minocula** [comment]
@PaulLereverend  Salue Paul, I've tried to use the great "NextcloudVideo_Converter". I use nextcloud 18.0.3 on my Docker envirement.  I've installed Ffmpeg. 
Now I want to use your recommendation  to setup the Ffmpeg $PATH. How do I do that? Can you give me a hint. Thx in advance and kind regards from switzerland
